app/controllers/api/users.php

```diff
@@ -1156,11 +1156,19 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
             /** @var Document $session */
             $dbForProject->deleteDocument('sessions', $session->getId());
             //TODO: fix this
+
+            $queueForEvents
+                ->setEvent('users.[userId].sessions.[sessionId].delete')
+                ->setParam('userId', $user->getId())
+                ->setParam('sessionId', $session->getId())
```

Comment:
Do we have any way to remove this param after the loop?

Reply:
Added `->reset()`